### PR TITLE
Refactors RangeInput and adds Storybook dark background

### PR DESCRIPTION
**Pull request by wassgha**
### Changes
- Refactors the `RangeInput` element to use the same implementation across both instances
- Adds a Storybook component for `RangeInput`
- Adds the ability to set Storybook backgrounds to a dark theme for light components

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
     "url": "https://github.com/google/web-stories-wp/issues"
   },
   "dependencies": {
+    "@storybook/addon-backgrounds": "^5.3.9",
```
Comment (swissspidy) [inline package.json]: Should be a dev dependency

File: assets/src/edit-story/components/canvas/gridview/index.js [hunk 2]
```diff
@@ -133,6 +84,8 @@ function RangeControl( { value, onChange } ) {
 				step="1"
 				value={ value }
 				onChange={ ( evt ) => onChange( Number( evt.target.value ) ) }
+				width="360px"
```
Comment (dvoytenko) [inline assets/src/edit-story/components/canvas/gridview/index.js]: I'd rather have this as `width={360}` and add `px` inside the component.

File: assets/src/edit-story/components/canvas/gridview/index.js [hunk 3]
```diff
@@ -133,6 +84,8 @@ function RangeControl( { value, onChange } ) {
...
+				thumbSize="24"
```
Comment (dvoytenko) [inline assets/src/edit-story/components/canvas/gridview/index.js]: Can we pass it as a number? IMHO that's easier to read.

File: assets/src/edit-story/components/rangeInput/index.js [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+	appearance: none;
+	width: ${ ( { thumbSize = 16 } ) => thumbSize }px;
+	height: ${ ( { thumbSize = 16 } ) => thumbSize }px;
+	background: #fff;
+	cursor: pointer;
+	border-radius: 50px;
+	box-shadow: 0 2px 5px rgba(0, 0, 0, 0.15);
+`;
+
+export default styled.input.attrs( ( { min, max, step, value, onChange = () => {} } ) => ( {
```
Comment (dvoytenko) [inline assets/src/edit-story/components/rangeInput/index.js]: Why default `onChange`?

**Review by dvoytenko — DISMISSED** [review]

File: .storybook/main.js [hunk 5]
```diff
@@ -25,5 +25,6 @@ module.exports = {
 		'@storybook/addon-knobs/register',
 		'@storybook/addon-storysource/register',
 		'@storybook/addon-viewport/register',
+		'@storybook/addon-backgrounds/register',
```
Comment (swissspidy) [inline .storybook/main.js:28]: This addon is a really nice feature 👌 makes a lot of sense

File: assets/src/edit-story/components/canvas/gridview/index.js (hunk 2 above)
Comment (wassgha) [reply assets/src/edit-story/components/canvas/gridview/index.js]: this is to allow for percentage values, if we don’t think that’s useful then sure

Comment (swissspidy) [reply assets/src/edit-story/components/canvas/gridview/index.js]: What about always using 100% and let the parent element dictate the max width?

**Review by swissspidy — APPROVED** [review]

**Pull request merged**